source/_docs/z-wave/device-specific.markdown

```diff
@@ -84,6 +84,22 @@ $ sudo usermod -a -G dialout homeassistant
 
 </div>
 
+### Razberry Board with Hass.io
+
+If you are using hass.io on a Raspberry Pi please note that the `/boot/`-folder is located in a FAT32-partition on the SD-card and will not be accessible from a mac/linux machine. To access the folder from a mac with the SD-card connected:
```

Comment:
Hass.io or HassOS? This is getting confusing pretty quickly.

Secondly, this only provides a method for Mac. Probably it is more wisely to document the use of the HassOS debug SSH access: <https://developers.home-assistant.io/docs/en/hassio_debugging.html>

The config file needed is in `/mnt/boot`

Reply:
I am a bit confused myself to be honest, but after some reading I believe that this is HassOS. Is that correct? 

The instructions are meant for people where updating the config.txt is needed and a USB stick is not at hand. Configuring the HassOS debug SHH access seems to require a USB stick as well.

If you plug the SD-card into a Windows PC, the partition should show up, since it is some form of Windows partition, right?

That leaves Mac/Linux. I guess the diskutil-commands are Mac-specific, but I can find corresponding commands for Linux. Any preferred way of documenting them side-by-side?